userspace/engine/rule_loader.cpp

```diff
@@ -326,19 +273,89 @@ static void resolve_macros(
 	}
 }
 
-// note: there is no visibility order between filter conditions and lists
-static shared_ptr<ast::expr> parse_condition(
-	string condition,
-	indexed_vector<rule_loader::list_info>& lists)
+struct macro_list_inserter
```

Comment:
If this is only for lists, why not naming this `list_inserter`? Macros and lists are different things in Falco

Reply:
renamed